apis/v1alpha1/event_bus.go

```diff
@@ -66,7 +66,7 @@ type EventBusStatus struct {
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:name="ARN",type=string,priority=0,JSONPath=`.status.ackResourceMetadata.arn`
-// +kubebuilder:printcolumn:name="SYNCED",type=string,priority=0,JSONPath=`.status.conditions[?(@.type=="ACK.ResourceSynced")].status`
+// +kubebuilder:printcolumn:name="Synced",type="string",priority=0,JSONPath=".status.conditions[?(@.type==\"ACK.ResourceSynced\")].status"
```

Comment:
Looks like we are using different characters here - does it all look in practice, @embano1 ?

Reply:
Nvm I see in the CRD definition that nothing actually changes.

Reply:
Yup, checked with `kubectl` output and it's all CAPS anyways